src/snowflake/snowpark/modin/pandas/index.py

```diff
@@ -0,0 +1,95 @@
+#
```

Comment:
I think this file should go in `_internal`. It should not be an API layer object.

Reply:
Why would it not go in the same location as `dataframe.py` or `series.py`? I am not sure if I fully understand this

Reply:
Ok moved it